mllib/src/main/scala/org/apache/spark/mllib/regression/StreamingLinearRegressionWithSGD.scala

```diff
@@ -59,6 +59,8 @@ class StreamingLinearRegressionWithSGD private[mllib] (
 
   val algorithm = new LinearRegressionWithSGD(stepSize, numIterations, miniBatchFraction)
 
+  var model: Option[LinearRegressionModel] = None
```

Comment:
I mean, write a getter/setter for `model` as a test. Right now it looks like you're just shadowing the member in the superclass? That doesn't seem right, like, this 'works' for the wrong reason.